express-post-json/server.js

```diff
@@ -0,0 +1,31 @@
+import express from 'express';
+
+const app = express();
+let nextId = 1;
+
+const grades = {};
+
+app.get('/api/grades', (req, res) => {
+  const array = [];
+  if (Object.keys(grades).length === 0) {
+    res.status(400).send('Currently no grades stored, please submit a grade.');
+  } else {
+    for (const obj in grades) {
+      array.push(grades[obj]);
+    }
+    res.send(array);
+  }
+});
+
+app.use(express.json());
```

Comment:
Typically we would put this at the top of the file, right after creating `app`. It's only required to put it before routes that need it, but it doesn't hurt to have it in there for everything, so typically we put them all at the top.